examples/linear_static.jl

```diff
@@ -0,0 +1,104 @@
+
+datadir = Pkg.dir("JuliaFEM", "examples", "linear_static")
+meshfile = joinpath(datadir, "JuliaFEMSMP18.med")
+mesh = aster_read_mesh(meshfile)
+
+# Next we will create the model and define Elasticity. Also elements are added
+# to the model.
+
+model = Problem(Elasticity, "OTHER", 3)
+model_elements = create_elements(mesh, "OTHER")
+
+# Elements need material properties and they are defined next
+
+update!(model_elements, "youngs modulus", 208.0E3)
+update!(model_elements, "poissons ratio", 0.30)
+update!(model_elements, "density", 7.80E-9)
+JuliaFEM.add_elements!(model, model_elements)
+
+# Let's ingore Seg3 and Tri6 elements using `filter`
+model.elements = collect(filter(x -> !isa(x, Union{Element{Seg3},
+                                            Element{Tri6}}), model.elements));
+
+# The whole idea of the JuliaFEM input is to be a normal Julia script, where the
+# user can freely define any functions needed to perform the task. Here we
+# define a function, which finds nodes on the given plane yz, xz or xy from the
+# given height.
+
+function find_nodes_at_certain_plane!(mesh,name,vector_id,distance,radius=6.0)
+    for (node, coords) in mesh.nodes
+        if isapprox(coords[vector_id], distance, atol=radius)
+            add_node_to_node_set!(mesh,name,node)
+        end
+    end
+end
+
+# We will find nodes from the xz-plane going through point (0,50,0) or actually
+# we previously defined the radius to be 6.0, which means (0,[44,56],0). In other
+# words we will select each node, which second coordinate value is between 44
+# and 56.
+find_nodes_at_certain_plane!(mesh,:mid_fixed,2,50.0)
+
+# We need to somehow handle the i's dot. I looked the rough coordinates of the
+# dot in FreeCAD and now we can search three closest nodes to these coordinates.
+# Those will be added to the same set `:mid_fixed`.
+
+ipoint = find_nearest_nodes(mesh,[165.0, 88.0, 10],3)
+for poi in ipoint
+    add_node_to_node_set!(mesh,:mid_fixed,poi)
+end
+
+# The fixed boundary conditions are defined next.
+
+fixed = Problem(Dirichlet, "fixed", 3, "displacement");
+fixed_elements = create_nodal_elements(mesh, "mid_fixed")
+JuliaFEM.add_elements!(fixed, fixed_elements)
+update!(fixed_elements, "displacement 1", 0.0)
+update!(fixed_elements, "displacement 2", 0.0)
+update!(fixed_elements, "displacement 3", 0.0)
+
+
+# Let's use simple acceleration load.
+update!(model_elements, "displacement load 1", 1.0)
+
+# Finally the ´Analysis` couples everything togeter.
+analysis = Analysis(Linear,model, fixed);
+
+# Let's request xdmf resuls
+add_results_writer!(analysis, Xdmf("model_results"; overwrite=true))
+
+# Now we have all we need to run the analysis.
+run!(analysis)
+
+# ## Postprocessing
+# This is how the stresses are requested
+push!(model.postprocess_fields,"stress")
+
+# Finally let's write the results to the xdmf (this is bug, the solver should
+# do this automatically)
+JuliaFEM.postprocess!(analysis,0.0)

```

Comment:
A matter of taste, but I would write here:
```julia
using JuliaFEM: postprocess!, write_results!
time = 0.0
postprocess!(analysis, time)
write_results!(analysis, time)
```

The reason why I explicitly define time is that the name of the variable clearly tells everyone what is the meaning of that 0.0 in the function argument.

Reply:
agree